pkg/config/config_test.go

```diff
@@ -920,6 +920,162 @@ func TestDeleteApplication(t *testing.T) {
 	}
 }
 
+func TestDeleteProject(t *testing.T) {
+	tempConfigFile, err := ioutil.TempFile("", "odoconfig")
```

Comment:
Can you use https://github.com/redhat-developer/odo/blob/master/pkg/testingutil/configs.go#L62 and https://github.com/redhat-developer/odo/blob/master/pkg/testingutil/configs.go#L117

Reference: https://github.com/redhat-developer/odo/blob/master/pkg/project/project_test.go#L34-L45

Reply:
I will get a import build error 
```
# github.com/redhat-developer/odo/pkg/config
FAIL	github.com/redhat-developer/odo/pkg/config [setup failed]
import cycle not allowed in test
package github.com/redhat-developer/odo/pkg/config (test)
	imports github.com/redhat-developer/odo/pkg/testingutil
	imports github.com/redhat-developer/odo/pkg/config
```